src/plugins/presentation_util/common/lib/test_helpers/test_styles.ts

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Should this be in a _fixtures_ folder, since it is only intended for using in tests?

Reply:
It seems to me, `test_helpers` are playing this role in `presentation_util`.